### ISSUE TITLE
commit lockfile

### ISSUE THREAD

**Issue by NotAShelf** [s0me1newithhand7s/myNixConf#9]
I do not know whether or not you intend for this configuration to be reproducible, but it makes almost no sense to *not* commit lockfiles.

Unless it somehow contains secrets , and from what I can tell, your flake.nix does not contain secrets, you should commit your flake.lock. This will allow for, e.g., nixos-rebuild or nix build to build from the exact same revision of your inputs and help improve the reproducibility of your system.

tl;dr: add your flake.lock to the repo

**Comment by s0me1newithhand7s** [comment]
hi fella!
this repo is not created for my use. this whole project aimed for those whp struggles with creating his own modular nixos config. 
i'm trying to make this config as open and forkable as i can (still WIP and not being tested :D), bc mine config in private repo and contains all i need to fully repoduce on both my machines from one flake.
thank you for being worried about!
